code/game/objects/effects/effect_system/effects_chem_smoke.dm

```diff
@@ -0,0 +1,123 @@
+/////////////////////////////////////////////
+// Chem smoke
+/////////////////////////////////////////////
+
+/obj/effect/particle_effect/chem_smoke
+	icon = 'icons/goonstation/effects/64x64.dmi'
+	icon_state = "smoke"
+	density = FALSE
+	opacity = FALSE
+	animate_movement = NO_STEPS
+	var/matrix/first = matrix()
+	var/matrix/second = matrix()
+	var/matrix/third = matrix()
+	var/first_scale = 0.1
+	var/second_scale = 5
+	var/third_scale = 2
+	var/spread_amount = 96
+
+/obj/effect/particle_effect/chem_smoke/New(location, chem_color)
+	..()
+	color = chem_color
```

Comment:
This has to be done here; if the color is applied after all the scaling and transform stuff are applied, the smoke will be colorless.